db/setup/schema.sql

```diff
@@ -70,3 +70,14 @@ CREATE TABLE
     "content" TEXT NOT NULL,
     "createdAt" TIMESTAMPTZ NOT NULL DEFAULT NOW ()
   );
+
+CREATE TABLE
+    IF NOT EXISTS ChatMessage (
+    "id" UUID DEFAULT gen_random_uuid () PRIMARY KEY,
+    "authorId" UUID REFERENCES Users (id) ON DELETE CASCADE,
+    "content" TEXT NOT NULL,
+    "createdAt" TIMESTAMPTZ NOT NULL DEFAULT NOW (),
+    "user1Id" UUID REFERENCES Users (id) ON DELETE CASCADE,
+    "user2Id" UUID REFERENCES Users (id) ON DELETE CASCADE,
+    CHECK ("user1Id" < "user2Id")
```

Comment:
The idea of this is to save a pair of users as a SET. This way we always identify unequivocally a conversation and simplify data retrieval.

We could store the pair of users in an intermediate relational table and have a `chatId` but would have to join both when fetching messages. Since this table is prone to grow a lot, better to avoid joins.

Reply:
Yes, my approach was a bit cumbersome, I'll update it. Thanks!